src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -63,6 +64,22 @@ class CreateProject extends React.Component {
     }
   }
 
+  componentDidMount() {
+    // TODO(jeff): Change key to AlertDefaultExperiment on the real experiment run
+    logExperiment({
+      organization: this.props.organization,
+      key: 'AlertDefaultExperimentTmp',
+      unitName: 'org_id',
+      unitId: parseInt(this.props.organization.id, 10),
+      param: 'exposed',
+    });
+    trackAnalyticsEvent({
+      eventKey: 'new_project.visit',
```

Comment:
We generally use `trackAdhocEvent` for higher volume events due to certain rate limits. Also, do you need to track this analytics event in addition to logging the experiment?

Reply:
Addressed in: https://github.com/getsentry/sentry/pull/17256